compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -5737,4 +5774,6 @@ THRIFT_REGISTER_GENERATOR(
     "    generated_annotations=[undated|suppress]:\n"
     "                     undated: suppress the date at @Generated annotations\n"
     "                     suppress: suppress @Generated annotations entirely\n"
-    "    unsafe_binaries: Do not copy ByteBuffers in constructors, getters, and setters.\n")
+    "    unsafe_binaries: Do not copy ByteBuffers in constructors, getters, and setters.\n"
+    "    annotations_as_metadata:\n"
+    "                     Include annotations as metadata in the generated code.\n")
```

Comment:
Just a tiny bit more clarification here would help I think.

```suggestion
    "                     Include Thrift field annotations as metadata in the generated code.\n")
```